packages/aws-cdk/README.md

```diff
@@ -388,6 +388,40 @@ $ cdk deploy --method=prepare-change-set --change-set-name MyChangeSetName
 For more control over when stack changes are deployed, the CDK can generate a
 CloudFormation change set but not execute it.
 
+#### Import existing resources
+
+You can pass the `--import-existing-resources` flag to the `deploy` command:
+
+```console
+$ cdk deploy --import-existing-resources
+```
+
+This allows you to automatically import the resources in your template that already
+exist in your AWS account during CloudFormation deployments.
+With this feature, you can reduce the manual effort of import operations and avoid
+deployment failures because of naming conflicts.
```

Comment:
```suggestion
Automatically import resources in your CDK application which represent
unmanaged resources in your account. 
Reduces the manual effort of import operations and avoids
deployment failures due to naming conflicts with unmanaged resources in your account
```

Reply:
adding some active voice to be more similar to how our other command readme's are written.